src/js/graphql_endpoint/Dockerfile

```diff
@@ -11,7 +11,7 @@ COPY package.json .
 COPY package-lock.json .
 
 RUN npm i
-
+RUN rm -rf node_modules/grpc/build/
```

Comment:
We can probably cache this by adding another build stage to this Dockerfile. E.g. add a section like: 
```dockerfile
FROM node:alpine3.10 AS grapl-graphql-endpoint-build-2
COPY from=grapl-graphql-endpoint-build node_modules node_modules
RUN rm -rf node_modules/grpc/build/
RUN mkdir -p /lambda/modules/
...
FROM node:alpine3.10 AS grapl-graphql-endpoint
...
COPY --from=grapl-graphql-endpiont-build-2 /lambda.zip /
...
```